client/my-sites/checklist/wpcom-checklist/index.js

```diff
@@ -87,6 +97,45 @@ class WpcomChecklist extends PureComponent {
 		}
 	};
 
+	handleSendVerificationEmail = e => {
+		e.preventDefault();
+
+		if ( this.state.pendingRequest ) {
+			return;
+		}
+
+		this.setState( {
+			pendingRequest: true,
+		} );
+
+		userLib.sendVerificationEmail( ( error, response ) => {
+			this.setState( {
+				emailSent: response && response.success,
+				error: error,
+				pendingRequest: false,
+			} );
+		} );
+	};
+
+	verificationTaskButtonText() {
+		const { translate } = this.props;
+		if ( this.state.pendingRequest ) {
+			return translate( 'Sending…' );
+		}
+
+		if ( this.state.error ) {
+			return translate( 'Error' );
+		}
+
+		if ( this.state.emailSent ) {
+			return translate( 'Email sent', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Email sent.' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/email-verification-checklist)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).